src/master-playlist-controller.js

```diff
@@ -369,10 +327,20 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     }
   }
 
-  useAudio(label) {
-    let audioEntry;
-    let newAudioPlaylistLoader;
-    let mediaGroupName = this.masterPlaylistLoader_.media().attributes.AUDIO;
+  useAudio() {
+    let media = this.masterPlaylistLoader_.media();
+    let master = this.masterPlaylistLoader_.master;
+
+    if (!media || !media.attributes || !media.attributes.AUDIO ||
```

Comment:
That said, we definitely need some guards (for instance, the !media.attributes for the case of the src URL being a playlist, and not a master)